실패율/main.py

```diff
@@ -0,0 +1,17 @@
+# 프로그래머스 실패율 문제 풀이
+# https://programmers.co.kr/learn/courses/30/lessons/42889
+
+def solution(N: int, stages: list):
+    # 실패한 사람 수를 리스트에 각 스테이지 마다 담음
+    fail_counts = [0 for i in range(0, N + 1)]
+    for stage in stages:
+        fail_counts[stage - 1] += 1
```

Comment:
리스트 컴프리헨션에서 의미 없는 변수는 `_` 로 사용하더군요!
8번줄까지 컴프리헨션 만들면 완전 파이썬 프로그래머 :+1: 
```suggestion
    fail_counts = [0 for _ in range(0, N + 1)]
    fail_counts = [fail_counts[stage - 1] += 1 for stage in stages]
```